src/icon-api.js

```diff
@@ -123,6 +123,8 @@ export class IconApi {
  * @type {Record<IconVariant['size'], number>}
  */
 const SIZE_AS_NUMERIC = {
+  // We're defaulting for medium size when size is unspecified, is this the correct approach??
+  size_unspecified: 2,
```

Comment:
is this the correct place to make this decision? does defaulting to medium makes sense?

Reply:
I think this is fine! Probably worth updating/removing the comment to remove the question.